tasks/bootlint.js

```diff
@@ -17,17 +21,44 @@ module.exports = function(grunt) {
     var options = this.options({
       stoponerror: false,
       stoponwarning: false,
-      showallerrors: false,
-      relaxerror: []
+      relaxerror: [],
+      reportXML: null,
+      reportCSV: null,
+      reportTAP: null
     });
 
     var totalErrCount = 0;
     var totalFileCount = 0;
+    var wcount = 0;
+    
+    
+    var reportXML = options.reportXML;
+    delete options.reportXML;
+
+    var reportCSV = options.reportCSV;
+    delete options.reportCSV;
+   
+    var reportTAP = options.reportTAP;
+    delete options.reportTAP;
+    
+      var output = '';
+    var xml ='<?xml version="1.0" encoding="utf-8"?>\r\n<checkstyle>\r\n';
+    var csv ='File|Line Number|Error Code|Severity\r\n';
+    var tap = "#Whole List Of BootLint Tests \r\n";
+    if (reportXML) {
+      grunt.util.hooker.hook(process.stdout, 'write', {
```

Comment:
`grunt.util.hooker` is deprecated. See: http://gruntjs.com/api/grunt.util#grunt.util.hooker